todo_app/app.py

```diff
@@ -1,15 +1,27 @@
-from flask import Flask
-
+from flask import Flask, render_template
+from werkzeug.utils import redirect
+from todo_app.data.session_items import *
 from todo_app.flask_config import Config
+from flask import request 
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    todoitems = get_items()
+    return render_template('index.html', todoitems = todoitems)
 
+@app.route('/add', methods=['POST'])
+def add():
+    if request.method == 'POST':
```

Comment:
We shouldn't need to run this check explicitly because you've specified the methods allowed in the app route - even if the else clause redirects to the index, you still won't hit it by visiting `http://127.0.0.1:5000/add` in your browser